lib/processors/resourceListCreator.js

```diff
@@ -138,11 +138,13 @@ module.exports = async function({resources, options}) {
 		mergedResources: DEFAULT_BUNDLE_RESOURCES_FILTER,
 		designtimeResources: DEFAULT_DESIGNTIME_RESOURCES_FILTER,
 		supportResources: DEFAULT_SUPPORT_RESOURCES_FILTER,
-		debugBundles: DEBUG_BUNDLES
+		debugBundles: DEBUG_BUNDLES,
+		dependencyResources: []
```

Comment:
Why make it part of the options and not of `parameters` itself (next to `resources`)?

Reply:
My idea was to keep the signature of having `resources` and `options` for processors, but it seems we already use other parameters in existing processors.
So having dependencyResources next to resources is probably better.

Reply:
Done